test/tests/reducer-flattrs-save.js

```diff
@@ -0,0 +1,100 @@
+"use strict";
```

Comment:
Suggestion: I noticed that this is almost an exact copy of test/tests/reducer-flattrs-submit.js so what about adding an abstraction so that we don't have so much duplicated code lying around?